components/admin_console/team_channel_settings/channel/details/channel_moderation.tsx

```diff
@@ -0,0 +1,347 @@
+                    />
+                </div>
+                <div>
+                    {disabledDueToCreatePosts &&
+                    <FormattedMessage
+                        id={formattedMessages[props.name][disabledDueToCreatePosts].id}
+                        defaultMessage={formattedMessages[props.name][disabledDueToCreatePosts].defaultMessage}
+                    />
+                    }
+                </div>
+                <div>
+                    {disabledKey &&
+                    <FormattedMessage
+                        id={formattedMessages[props.name][disabledKey].id}
+                        defaultMessage={formattedMessages[props.name][disabledKey].defaultMessage}
+                    />
+                    }
+                    {' '}
+                    {disabledKey &&
+                    <Link
+                        to={`${formattedMessages[disabledLinkKey].link}${disabledLinkKey === 'team_scheme' ? props.teamScheme!.id : ''}`}
+                    >
+                        {disabledLinkKey === 'team_scheme' && `${props.teamScheme!.display_name} `}
+                        <FormattedMessage
+                            id={`${formattedMessages[disabledLinkKey].id}`}
+                            defaultMessage={`${formattedMessages[disabledLinkKey].defaultMessage}`}
+                        />
+                    </Link>
+                    }
+                </div>
+            </td>
+            <td>
+                {!isNil(props.guests) &&
+                    <button
+                        className={classNames(
+                            'checkbox',
+                            {
+                                checked: props.guests.value && !isGuestsDisabled,
+                                disabled: isGuestsDisabled,
+                            }
+                        )}
+                        onClick={() => props.onClick(props.name, 'guests')}
+                        disabled={isGuestsDisabled}
+                    >
+                        {props.guests.value && !isGuestsDisabled && <CheckboxCheckedIcon/>}
+                    </button>
+                }
+            </td>
+            <td>
+                {!isNil(props.members) &&
+                    <button
+                        className={classNames(
+                            'checkbox',
+                            {
+                                checked: props.members.value && !isMembersDisabled,
+                                disabled: isMembersDisabled,
+                            }
+                        )}
+                        onClick={() => props.onClick(props.name, 'members')}
+                        disabled={isMembersDisabled}
+                    >
+                        {props.members.value && !isMembersDisabled && <CheckboxCheckedIcon/>}
+                    </button>
+                }
+            </td>
```

Comment:
Minor bit of duplication between the above two `<td>`'s. Could be moved into a function if you wanted.

Reply:
As discussed, a little bit of duplication shouldn't be a big problem in this case.